### PR TITLE
Welcome patient view

### PR DESCRIPTION
**Pull request by Karl-Heinrich**
- React router
- Adds a first patient view as root
- Admin view has moved to HOST/admin
- Endpoint suggestion for patient view in backend

digitales-wartezimmer-ipad

### CODE REVIEW

**Comment by yogan** [comment]
Had a very quick glance, looks awesome! Pretty much what I had in mind as well, glad I didn't start and rather wasted my time with [some Docker and npm shit](56).

Can offer to take a proper look tomorrow, but if someone is quicker (and I suppose so, it's late right now, and I might sleep late), go ahead and merge! 👈😎👈

File: docs/api.md [hunk 1]
```diff
@@ -109,13 +109,14 @@ Zustand der Warteschlange abrufen
...
 
-Zustand der Warteschlange über place-id und ticketNumber abfragen
+Zustand der Warteschlange über place-id und ticked-id abfragen
 -----------------------------------------------------------------
 
-> WEBSOCKET
-> HTTP-GET: ws:HOST/places/**\<place-id\>**/**\<ticket-number>**
+Ticket ID und Place ID erhält der Patient von der Praxis.
+
+> HTTP-GET: /places/**\<place-id\>**/**\<ticket_id\>**
```
Comment (yogan) [inline docs/api.md]:
```suggestion
> HTTP-GET: /places/**\<place-id\>**/**\<ticket-id\>**
```

File: frontend/src/index.tsx [hunk 2]
```diff
@@ -48,9 +50,15 @@ const store = createStore(fakeReducer, applyMiddleware(backendMiddleware));
...
 
-ReactDOM.render(<Provider store={store}>
-  <App />
-</Provider>, document.getElementById('root'));
+ReactDOM.render(
+  <Provider store={store}>
+    <Router>
+      <Switch>
+        <Route path="/" component={PatientWelcome} />
+        <Route path="/admin" component={App} />
```
Comment (yogan) [inline frontend/src/index.tsx]: No idea why, but when I navigate to /admin in the browser, I still see the PatientWelcome page. Can someone confirm?

Comment (t-animal) [reply frontend/src/index.tsx]: Can confirm.

**Comment by t-animal** [comment]
* I thought our MVP was that we don't count down any more, but only show the queues, as we don't know how to prioritize them and that's done by the doctor's assistant? If that's correct then we don't need the input field at all, we just show the queues. 

* I don't like the hard coded doctor's waiting room theme too much. Maybe we can make this easily configurable? I kinda like @ohaz 's  idea that this tool could be used by shops, too, to prevent too many people entering.

File: frontend/src/index.tsx (hunk 2 above)
Comment (migerh) [reply frontend/src/index.tsx]: Need to switch the two Routes, then it works. `/` matches `/admin`, too.

Comment (Karl-Heinrich) [reply frontend/src/index.tsx]: Changed this last minute. Thanks for finding the issue!

File: docs/api.md (hunk 1 above)
Comment (Karl-Heinrich) [reply docs/api.md]: Ups thanks!

**Pull request merged**